Source/Atomic/UI/UI.h

```diff
@@ -100,6 +100,9 @@ class UI : public Object, private tb::TBWidgetListener
     void ShowDebugHud(bool value);
     void ToggleDebugHud();
 
+    // Cycle debug HUD between showing primitive stats, current mode, profiler data, all three or none
+    void CycleDebugHudMode();
+
```

Comment:
Ah, we avoid /// because it confuses MS's auto commenting. But then that's for in-code comments, no method descriptions